apps/wpcom-block-editor/src/calypso/features/gutenboarding-editor-overrides.js

```diff
@@ -54,3 +58,65 @@ function updateSettingsBar() {
 		settingsBar.prepend( saveButton );
 	} );
 }
+
+function showFirstTimeNotice() {
+	const getPreviewButton = () => document.querySelector( '.editor-post-preview' );
+
+	const awaitPreviewButton = setInterval( () => {
+		const previewButton = getPreviewButton();
+		if ( ! previewButton ) {
+			return;
+		}
+		clearInterval( awaitPreviewButton );
+
+		// Create preview link
+		// Reuse the href from existing Preview button in the header
+		// and trigger the click event handler (registered through jQuery)
+		// as seen in handlePreview() in iframe-bridge-server.js.
+		const previewLink = (
+			<a
+				href={ previewButton.href }
+				onClick={ ( e ) => {
+					e.preventDefault();
+					// Need to requery this element because the PreviewButton
+					// gets recreated when window is resized hence referencing
+					// an old destroyed button wouldn't work.
+					getPreviewButton().click();
+				} }
+				target="_top"
+			/>
+		);
+
+		// Create first time notice
+		const blockEditorHeader = document.querySelector( '.block-editor-editor-skeleton__header' );
+		const firstTimeNotice = document.createElement( 'div' );
+
+		// It seems that the close button doesn't work when the
+		// notice component is manually injected, so we'll handle
+		// the removal of this notice manually.
+		const handleRemove = () => {
+			blockEditorHeader.removeChild( firstTimeNotice );
+		};
+
+		ReactDOM.render(
+			<Notice
+				className="gutenboarding-editor-overrides__first-time-notice"
+				status="success"
+				onRemove={ handleRemove }
+			>
+				{ createInterpolateElement(
```

Comment:
There might be older Gutenberg versions on Atomic sites that don't support this function;

Let's add `type of createInterpolateElement === 'function'` check so safeguard. I think we can just disable this for those sites; they're very much minority.

Reply:
Added.